init.js

```diff
@@ -13,7 +12,7 @@ if (process.env.NODE_ENV === 'production') {
 
 throng({ workers: 1, lifetime: Infinity }, () => {
 	const loadServer = () => {
-		return require('./server/server').startServer();
+		return require('./dist/server/server/server').startServer();
```

Comment:
**Important change** — there are many path rewrites, but I just wanted to point out how silly this one is :joy:.

Reply:
😳